orcid-web/src/main/resources/freemarker/includes/ng2_templates/notification-body-ng2-template.ftl

```diff
@@ -3,7 +3,7 @@
         <div class="notifications-inner">
             <!--AMENDED-->
             <div *ngIf="notification.notificationType=='AMENDED'">
-                <div *ngIf="TOGGLZ_VERBOSE_NOTIFICATIONS; else defaultNotifications">
```

Comment:
Hey @auumgn look at this .... `else defaultNotifications` means that the `ng-template`  defaultNotifications will be displayed if the ngIf condition is false; so, please go ahead and remove this ng-template as well:


https://github.com/ORCID/ORCID-Source/pull/6311/files#diff-77a64175978df96cacd724d668cc06db330a99231ec3786e5d662952cef49ccbR122